runners/google-cloud-dataflow-java/worker/src/main/java/org/apache/beam/runners/dataflow/worker/windmill/client/grpc/observers/DirectStreamObserver.java

```diff
@@ -39,7 +39,9 @@
 @ThreadSafe
 public final class DirectStreamObserver<T> implements StreamObserver<T> {
```

Comment:
maybe this change can be submitted separately? Would be nice to have a test for it showing the previous bug as well.